PyInstaller/loader/rthooks/pyi_rth_gstreamer.py

```diff
@@ -19,3 +19,6 @@
 
 # Tested on OSX only.
 os.environ['GST_PLUGIN_PATH'] = os.path.join(sys._MEIPASS, 'gst_plugins')
+
+# only use packaged plugins
```

Comment:
Would you please add a note, why this is necessary or useful or which problem this solves. Thanks.